src/test/java/com/redhat/cloud/policies/app/model/validation/ActionValidatorTest.java

```diff
@@ -3,15 +3,15 @@
 import javax.validation.ConstraintValidatorContext;
 
 import com.redhat.cloud.policies.app.EnvironmentInfo;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
If Checkstyle was used in that project, that change would cause a build failure (wrong imports order).